.gitignore

```diff
@@ -48,3 +48,7 @@ coverage.json
 
 #tenderly config 
 tenderly.yaml
+
+MultiSigWallet
+MultiSigWalletOverride/ganache_data/
+plasma_framework/vault_authority
```

Comment:
might be redundant if we with this PR: https://github.com/omgnetwork/plasma-contracts/pull/679?